utils/rtty/files/rtty.init

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+
+BIN=/usr/sbin/rtty
+
+start_rtty() {
+    local cfg="$1"
+    local interface ifname id description host port ssl
+
+    uci_validate_section rtty rtty "${1}" \
+        'interface:uci("network", "@interface"):br-lan' \
+        'id:maxlength(63)' \
+        'description:maxlength(126)' \
```

Comment:
Was it intended to be `maxlength(127)` instead of `126`?   I am not quite sure about this one

Reply:
My program is a limit 126 character.